index.html

```diff
@@ -229,5 +229,46 @@ <h2>Skills</h2>
 				</ul>
 			</div>
 		</section>
+
+		<section class="form">
+			<h2>Contact me</h2>
+			<p>
+				If you have an application you are interested in developing, a feature
+				that you need built or a project that needs coding. I’d love to help
+				with it
+			</p>
+			<form action="https://formspree.io/f/mqknezaw" method="POST">
+				<label>
+					<input
+						class="sm-input"
+						required
+						maxlength="30"
+						placeholder="Your Name"
+						type="text"
+					/>
+				</label>
+				<label>
+					<input
+						class="sm-input"
+						placeholder="youremail@here.com"
+						required
+						type="email"
+						name="_replyto"
+					/>
+				</label>
```

Comment:
- [ ] You've implemented the features of the form well. Please ensure you are following the Figma design as closely as possible. If you notice the font you are using does not match the Figma
## Your Design
![image](https://user-images.githubusercontent.com/18730106/149019630-45baad5c-e222-4077-b7be-e8254cad5432.png)

## Figma Design
![image](https://user-images.githubusercontent.com/18730106/149019656-f5f04456-4ea2-4359-8c3d-c1524dea3760.png)

The correct `font-name` property should be Poppins, while you are using Tahoma.